util/common/agent_util_unix.go

```diff
@@ -180,16 +180,22 @@ func StopAgent() {
 	log.Printf("Agent is stopped")
 }
 
-func ReadAgentOutput(d time.Duration) string {
+func ReadAgentLogfile(logfile string) string {
```

Comment:
I find it easier to work with the log file than journald for debugging issues in my env. I can change this back if need be would rather not.